sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -133,6 +133,21 @@ export interface ServiceBusAdministrationClientOptions extends PipelineOptions {
 export type EntitiesResponse<T extends object> = WithResponse<Array<T>> &
   Pick<PageSettings, "continuationToken">;
 
+/**
+ * @internal
```

Comment:
If we are not exporting this function from this file, then I dont believe we need to add the `internal` tag here...